File: eform-client/src/app/plugins/modules/items-group-planning-pn/models/list/items-list-case-pn.model.ts
export class ItemsListCasePnModel {
  total: number;
  items: Array<ItemsListPnItemCaseModel> = [];
}

export class ItemsListPnItemCaseModel {
  id: number;
  date: string;
  name: string;
  description: string;
  itemNumber: string;
  locationCode: string;
  buildYear: string;
  type: string;
  location: string;
  status: number;
  fieldStatus: string;
  comment: string;
  numberOfImages: number;
  sdkCaseId: number;
  sdkeFormId: number;
}

File: eFormAPI/Plugins/ItemsGroupPlanning.Pn/ItemsGroupPlanning.Pn/Infrastructure/Models/ItemsListCasePnModel.cs
namespace ItemsGroupPlanning.Pn.Infrastructure.Models
{
    using System.Collections.Generic;

    public class ItemsListCasePnModel
    {
        public int Total { get; set; }
        public List<ItemsListPnItemCaseModel> Items { get; set; }
            = new List<ItemsListPnItemCaseModel>();
    }
}